tasks/feature_spec.md

```
@@ -124,3 +124,294 @@ MCP Patch `create_chart` major chart support (Phase 1)
 - `drafts` に新規 `.xlsx` が生成される
 - 8種類すべての `create_chart` オペレーションが成功する
 - エラーが返らない
+
+## Feature Name
```

markdownlint-cli2 (0.21.0)

[warning] Multiple headings with the same content

(MD024, no-duplicate-heading)